src/ng-select/lib/ng-select.component.ts

```diff
@@ -86,6 +86,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
     @Input() addTagText: string;
     @Input() loadingText: string;
     @Input() clearAllText: string;
+    @Input() appearance: string;
```

Comment:
add this new input to the readme, and point that it only controls material theme appearance at the time

Reply:
README.md updated